server/build.gradle

```diff
@@ -409,6 +409,7 @@ tasks.register("japicmp", me.champeau.gradle.japicmp.JapicmpTask) {
     failOnModification = true
     ignoreMissingClasses = true
     annotationIncludes = ['@org.opensearch.common.annotation.PublicApi', '@org.opensearch.common.annotation.DeprecatedApi']
+    annotationExcludes = ['@org.opensearch.common.annotation.InternaApi']
```

Comment:
@andrross closing the loop here with Java API compatibility checks (complementing https://github.com/opensearch-project/OpenSearch/pull/14575), thanks!